varada_trino_manager/main.py

```diff
@@ -342,6 +343,35 @@ def runner(jsonpath, concurrency, random, iterations, queries_list):
                  queries_list=[q_series.split(',') for q_series in queries_list], con=con)
 
 
+@option("-d", "--destination-dir", type=ClickPath(), default=None, help="Destination dir to save the json")
+@argument("query_id", nargs=1)
+@query.command()
+def json(query_id, destination_dir):
+    """
+    Get query json by query_id
+    Where query_id is the unique Trino Query Id, format example: 20210513_063641_00004_raiip
+    """
+    destination_dir = destination_dir if destination_dir else Paths.logs_path
+    con = get_config().get_connection_by_name("coordinator")
+    with PrestoRest(con=con) as trino_rest:
```

Comment:
same with block + isn't it repeating? consider making a helper function and call it